generate-xml/domain.nix

```diff
@@ -174,9 +174,9 @@ let
           [
             (subelem "model"
               [
-                (subattr "fallback" typeInt)
+                (subattr "fallback" typeString)
               ]
-              [ (subelem name [ ] typeString) ]
+              [ (subelem "name" [ ] typeString) ]
```

Comment:
OK, could you change `subelem` to `sub`? It turns out `model` doesn't have a `name` subelement.